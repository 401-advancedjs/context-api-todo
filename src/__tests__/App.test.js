import React from 'react';
import App from '../component/App/App.js';
import renderer from 'react-test-renderer';

describe('<App />', () => {
  test('basic rendering', () => {
    const mountedHeader = shallow(<App />);
    expect(mountedHeader.find('App')).toBeTruthy();
  });

  test('rendering follows the snapshot', () => {
    const snapshot = renderer.create(<App />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});