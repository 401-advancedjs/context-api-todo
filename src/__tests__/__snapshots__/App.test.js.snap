// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> rendering follows the snapshot 1`] = `
<section
  className="todo"
>
  <div>
    <h2>
      There are 
      0
       items in the list
    </h2>
  </div>
  <div>
    <form
      onSubmit={[Function]}
    >
      <input
        onChange={[Function]}
        placeholder="Add To Do List Item"
      />
    </form>
  </div>
  <div>
    <ul />
  </div>
</section>
`;
